ACMAS/app/ACMAS_Web/views.py

```diff
@@ -142,7 +127,6 @@ def uploadOptions(request):
     return render(request, "upload-options.html")
 
 
-@csrf_protect
```

Comment:
is there a reason you took off the csrf flags here?

Reply:
They started causing issues with how the URL was being displayed. I removed them for the time being so that I have more time to investigate this singular issue in a contained setting separate from the massive deployment change.